resource/file/file.go

```diff
@@ -214,3 +165,33 @@ func Contains(f resource.Filer, contents ...*regexp.Regexp) error {
 		return fmt.Errorf("%s does not match any provided regular expression", f)
 	})
 }
+
+// Md5 checks if the provided file md5 checksum is the same as the one passed in as paramter
+// It returs error if the provided file can't be opened
+func Md5(f resource.Filer, sum string) error {
```

Comment:
How about `IsMd5Sum` to be consistent with the rest of the functions in this resource ?